app/src/main/java/com/internship/move/ui/onboarding/OnboardingFragment.kt

```diff
@@ -43,14 +42,12 @@ class OnboardingFragment : Fragment(R.layout.fragment_onboarding), SkipCallback
     }
 
     override fun onSkipButtonClick() {
-        updateSharedPreferences()
+        notifyHasViewedOnboarding()
         findNavController().navigate(OnboardingFragmentDirections.actionOnboardingFragmentToAuthenticationGraph())
     }
 
-    private fun updateSharedPreferences() {
-        requireActivity().getSharedPreferences(KEY_APP_PREFERENCES, MODE_PRIVATE).edit()
-            .putBoolean(KEY_HAS_VISITED_ONBOARDING, true)
-            .apply()
+    private fun notifyHasViewedOnboarding() {
```

Comment:
I would rename this method to something more appropriate for a setter. This is because you're not notifying anything, you're just setting a value. This way it would be more suggestive of its effect

Reply:
Same thing for the viewModel method itself